pkg/api/v1/types.go

```diff
@@ -354,6 +371,11 @@ const (
 	// used for PersistentVolumes where the bound PersistentVolumeClaim was deleted
 	// released volumes must be recycled before becoming available again
 	VolumeReleased PersistentVolumePhase = "Released"
+	// used for PersistentVolumes that have been recycled and can be made available
+	// again to new PersistentVolumeClaims
+	VolumeRecycled PersistentVolumePhase = "Recycled"
```

Comment:
It's a different model. This is fine for now, but Phase has big problems.

For example, this PR is a breaking change. Technically, we shouldn't ever add enum values to existing API versions.

Also, you're driving behavior off the phase. Phase was intended to be a summary of the observed status, not a state machine.